src/ui/FileViewer/ImagesView.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+import { isImage } from '../../utils';
+import Label, { LabelTypography, LabelColors } from '../Label';
+
+const BUTTON_ICON_SIDE_LENGTH = '24px';
+
+export function ImagesView({ props, stringSet }: {
+  props: FileViewerComponentProps,
+  stringSet?: any,
```

Comment:
Use - `useContext(LocalizationContext )`

Reply:
removed it.